### PR TITLE
Add redhat-appstudio/integration-service repository

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  to: redhat-appstudio-integration-service-image
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml:14]: probably better to use 4.11 or 4.12 (which should be backward compatible)

File: ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      memory: 200Mi
+tests:
+- as: integration-service-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml:30]: switching to 4.12 here would probably make sense as well (although I think we're using 4.11 in other jobs - but it should be updated)

**Review by psturc — APPROVED** [review]
just minor comments, otherwise lgtm

File: ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml (hunk 2 above)
Comment (jinqi7) [reply ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml]: Updated. Thanks

File: ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml (hunk 1 above)
Comment (jinqi7) [reply ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml]: Thanks. Updated.

**Review by dirgim — APPROVED** [review]
/lgtm

**Pull request merged**